tests/fixtures.py

```diff
@@ -761,6 +761,47 @@ def parametrize(params, values, **kwargs):
     key=lambda t: (t[0], t[1]),
 )
 
+# this list should be merged into the one above (FEATURE_PRESENSE_TESTS)
+# once the debug symbol functionality has been added to all backends
+FEATURE_SYMTAB_FUNC_TESTS = [
+    (
+        "2bf18d",
+        "function=0x4027b3,bb=0x402861,insn=0x40286d",
+        capa.features.insn.API("__GI_connect"),
+        True,
+    ),
+    (
+        "2bf18d",
+        "function=0x4027b3,bb=0x402861,insn=0x40286d",
+        capa.features.file.FunctionName("__GI_connect"),
+        True,
+    ),
```

Comment:
as described above, `FunctionName` is a function-scoped feature that should be applied to the address of the function that *is* `___GI_connect`

Reply:
these tests still need to be updated.